modules/cnf-topology-aware-lifecycle-manager-operator-troubleshooting.adoc

```diff
@@ -34,7 +34,7 @@ metadata:
   namespace: operator-namspace
 # ...
 spec:
-  source: redhat-operators-v2 <1>
+  source: disconnected-redhat-operators-v2 <1>
```

Comment:
Can this be redhat-operators-disconnected so that it matches the reference:
https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml#L87
Thank you!

Reply:
Thanks @imiller0 - I've updated the instances accordingly

Reply:
@imiller0 - Any other issues or can I get this update merged? Thanks